ui/src/cohortsSlice.ts

```diff
@@ -41,13 +41,33 @@ const cohortsSlice = createSlice({
           cohortId,
           group: {
             id: generateId(),
+            name: "",
```

Comment:
No need to fill in a default value now that it's optional.

Reply:
omg.... i just merged and have not noticed this comment. What should I do now....